src/pages/known-issues/index.md

```diff
@@ -0,0 +1 @@
+#
```

Comment:
@kareemmyk  We need to evaluate this doc https://developer.adobe.com/photoshop/uxp/2022/uxp-api/known-issues/ and determine if they are applicable to indesign